src/OpenTelemetry.Instrumentation.AWSLambda/OpenTelemetry.Instrumentation.AWSLambda.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>AWS Lambda tracing wrapper for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);AWS Lambda</PackageTags>
     <MinVerTagPrefix>Instrumentation.AWSLambda-</MinVerTagPrefix>
+    <EnablePublicApi>true</EnablePublicApi>
```

Comment:
You might want to enable code analysis as well.

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj#L9

Reply:
That is disabled by default? Weird, how should I know 😅
I will try to enable it.
Maybe it would be better to enable it by default and explicitly disable it in projects where it causes problems?

Reply:
I tried it locally, the tool only suggested adding some ConfigureAwait(false) in some places. I will submit a PR for that eventually, but I don't want to hold up the release longer, so I will do that after #590